pennylane_lightning/core/src/simulators/lightning_kokkos/gates/BasicGateFunctors.hpp

```diff
@@ -372,6 +372,25 @@ void applySWAP(Kokkos::View<Kokkos::complex<PrecisionT> *> arr_,
         });
 }
 
+template <class ExecutionSpace, class PrecisionT>
+void applyISWAP(Kokkos::View<Kokkos::complex<PrecisionT> *> arr_,
+               const std::size_t num_qubits,
+               const std::vector<std::size_t> &wires,
+               [[maybe_unused]] const bool inverse = false,
```

Comment:
If `inverse` is unused, does it mean ISWAP is an involutory matrix?

Reply:
ISWAP is not a involutory matrix. Its adjoint is not the same to itself. If inverse is true, that indicates to apply its adjoint. It requires additional code to handle that situation, which is missing here.